src/QsCompiler/Core/Transformations/CommonTransformationItems.fs

```diff
@@ -0,0 +1,38 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.QsCompiler.Transformations.Core
+
+open Microsoft.Quantum.QsCompiler.DataTypes
+open Microsoft.Quantum.QsCompiler.SyntaxTree
+
+
+/// <summary>
+/// Contains virtual methods that are invoked
+/// by several different kinds of nodes in the syntax tree.
+/// </summary>
+type CommonTransformationItems internal () =
```

Comment:
I renamed it to CommonTransformationNodes. CommonTransformation seems odd to me since it is not fully self contained transformation but rather a collection of nodes shared by all transformations.